### DISCUSSION TITLE
Keywords Python SDK

### DISCUSSION THREAD

**Discussion by YupChutLuke** [deepgram/community#698] [General help] [answered]
I am working with the prerecorded transcribe. I would like to specify some Proper Nouns for the model to listen to. However, I am running into an issue and I can't tell if there is a way to do this through the Python SDK

Currently I am working with something like this
```
 options = PrerecordedOptions(
        model='nova-2-phonecall',
        smart_format=True,
        keywords=["<PROPER_NOUN>", ],
        
    )
```

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @YupChutLuke, that code sample looks correct to me. You can pass a keyword with `keywords=["<PROPER_NOUN>"]`, or add an intensifier value with `keywords=["<PROPER_NOUN>:2"]` (the default intensifier value is 1). Then you can add as many keywords as you need within that list.

Are you getting an error with your code, or are the keyword boosting results not what you're expecting?
